test/utils/withConsole.ts

```diff
@@ -1,11 +1,22 @@
-import mockConsole from "jest-mock-console"
+import mockConsole, { MockObj } from "jest-mock-console"
 
-export default function withConsole(settings, fn) {
-    if (typeof settings === "function") {
-        fn = settings
-        settings = undefined
+export function withConsole(fn: Function): void
+export function withConsole(settings: MockObj, fn: Function): void
+export function withConsole(props: Array<ConsoleProps>, fn: Function): void
+
+export function withConsole(...args: Array<any>): void {
```

Comment:
![image](https://user-images.githubusercontent.com/12650339/73654008-74855300-463f-11ea-8f41-4bc14ef99414.png)

Reply:
Well, I think if you declare those overloads first and then `export default withConsole` it would work as well, but personally I like named exports too, so I am fine with this.

Reply:
Okay.  I'll leave this as is, using the overloads and named exports.